sys/include/dbgpin.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_dbgpin Direct pin control for debugging/profiling
+ * @ingroup     sys
+ *
+ * @warning     This module does not verify the given pin number, so make sure
+ *              the pin numbers you use are actually configured!
+ *
+ * @{
+ * @file
+ * @brief       GPIO wrapper for debugging/profiling purposes
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef DBGPIN_H
+#define DBGPIN_H
+
+#include "kernel_defines.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#ifndef DBGPIN_PINS
+#error Please specify the pins to use with the dbgpin module (DBGPIN_PINS)
+#endif
+
+extern const gpio_t dbgpin_pins[];
+
+static inline void dbgpin_set(unsigned pin)
+{
+    gpio_set(dbgpin_pins[pin]);
+}
+
+static inline void dbgpin_clr(unsigned pin)
```

Comment:
```suggestion
static inline void dbgpin_clear(unsigned pin)
```

Better keep the spelling consistent with `gpio_clear()`

Reply:
I personally like the shorter notation better. Any other opinions?

Reply:
I think I agree with @maribu here, not something I would insist upon but I think consistency is better.

Reply:
For the shorter notation has a practical benefit, as I tend to write these statements all over the place again and again when debugging, so short function names saved some typing.

But 2 against 1, I am official outvoted :-) So I will adapt those names.

Reply:
addressed